src/docs/reference/flutter-cli.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "flutter: The Flutter command-line tool"
+description: "The reference page for using 'flutter' in a terminal window."
+---
+
+The `flutter` command-line tool is how developers (or IDEs on behalf of developers) interact with Flutter.
```

Comment:
Long line again. (I'm sure there's a better way to do it, but sometimes I'll open the file in Sublime with an 80-char line limit, and then eyeball the line numbers on the left for gaps indicating that the line had to wrap.)